mle/utils/system.py

```diff
@@ -75,7 +75,15 @@ def check_config(console: Optional[Console] = None):
     """""
     console = console or Console()
     current_work_dir = os.getcwd()
-    config_path = os.path.join(current_work_dir, 'project.yml')
+
+    config_dir = os.path.join(current_work_dir, '.mle')
+    config_path = os.path.join(config_dir, 'project.yml')
+
+    # move the old config file to the new path for compatibility (delete in future)
```

Comment:
to keep the compatibility for the previous project, the old config file will be moved into new paths